### PR TITLE
Allow to pass max_size attribute

### PR DESCRIPTION
**Pull request by avokhmin**
Example:

```
foo.slim

= transloadit :bar, max_size: 10.megabytes
```

### CODE REVIEW

**Comment by avokhmin** [comment]
PS: https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/

> After February 23 2014, we will only provide security fixes for 1.9.3 until February 23 2015, after which all support will end for 1.9.3.

**Comment by recurser** [comment]
+1

**Comment by rmehner** [comment]
Thanks, looks good :)

> PS: https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/
> 
> After February 23 2014, we will only provide security fixes for 1.9.3 until February 23 2015, after which all support will end for 1.9.3.

Yep, we know, but then again there's no need for us to push to 2.x or whatever, as long as we don't have any security issue from our side or have to jump hoops to maintain that compatibility, there's no reason to not test on 1.9.x as well and push newer version onto our users :)

**Comment by rmehner** [comment]
1.2.0 has been pushed, thanks again!

File: lib/transloadit/rails/engine.rb [hunk 1]
```diff
@@ -45,25 +45,38 @@ def self.configuration
...
       end
 
       #
       # Creates an assembly for the named template.
       #
+      # name    - The String or Symbol template name.
+      # options - The Hash options used to refine the Assembly (default: {}):
+      #           :steps    - The Hash with Assembly Steps (optional).
+      #           :max_size - The Integer maximum size an upload can have in bytes (optional).
       def self.template(name, options = {})
```
Comment (avokhmin) [inline lib/transloadit/rails/engine.rb]: > Yep, we know, but then again there's no need for us to push to 2.x or whatever, as long as we don't have any security issue from our side or have to jump hoops to maintain that compatibility, there's no reason to not test on 1.9.x as well and push newer version onto our users :)

yeah, but `def self.template(name, max_size: nil, **options)` looks better :)

**Comment by avokhmin** [comment]
> 1.2.0 has been pushed, thanks again!

Thanks!

File: lib/transloadit/rails/engine.rb (hunk 1 above)
Comment (rmehner) [inline lib/transloadit/rails/engine.rb]: > yeah, but def self.template(name, max_size: nil, **options) looks better :)

I absolutely agree with you, but this is a small price to pay to be compatible with older versions :) Thanks again for your pull request :)

**Comment by tim-kos** [comment]
Thank you @avokhmin !

**Pull request merged**